devel/ldid/Portfile

```diff
@@ -1,38 +1,47 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
+PortGroup           cmake 1.0
 PortGroup           github 1.0
+PortGroup           openssl 1.0
 
-name                ldid
 github.setup        sbingner ldid 5b8581c
 version             2.1.5
-revision            0
+revision            1
```

Comment:
why increase the `revision`; does it change the installed files?

Reply:
It should. The port has a history for having a symlink called `ldid2`, but I just realized I forgot to add it.